messages/npm.dependencies.hardcode.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "lock a list of dependencies to a target release or default to 'latest', place these entries in 'packagesToHardcodeVersions' entry in the package.json",
+  "flags": {
+    "release": "The name of the release you want, e.g. 'latest-rc', or 'latest'",
```

Comment:
I think this should be `tag` instead of `release`? Thoughts?

Reply:
yes - I was completely blanking on that word earlier